src/Kunstmaan/UserManagementBundle/Controller/UsersController.php

```diff
@@ -197,57 +196,55 @@ public function editAction(Request $request, $id)
                 return new RedirectResponse(
                     $this->generateUrl(
                         'KunstmaanUserManagementBundle_settings_users_edit',
-                        array('id' => $id)
+                        ['id' => $id]
                     )
                 );
             }
         }
 
-        $params = array(
+        $params = [
             'form' => $form->createView(),
             'user' => $user,
-        );
+        ];
 
         if ($tabPane) {
-            $params = array_merge($params, array('tabPane' => $tabPane));
+            $params = array_merge($params, ['tabPane' => $tabPane]);
         }
 
         return $params;
     }
 
     /**
-     * Delete a user
-     *
-     * @param Request $request
-     * @param int     $id
-     *
-     * @Route("/{id}/delete", requirements={"id" = "\d+"}, name="KunstmaanUserManagementBundle_settings_users_delete", methods={"POST"})
```

Comment:
Do you have any idea how we could avoid BC breaking but still solving the issue until we remove the unsafe method? Otherwise I might have to do some custom work for our project since the client thinks this is quite important.